src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Switch, Route } from "react-router-dom";
+
+import { Header } from "./layout";
+import { Home, NotFound } from "./pages";
+
+const App = () => {
+  return (
+    <>
+      <Header />
+      <Switch>
+        <Route exact path="/">
+          <Home />
+        </Route>
```

Comment:
would more pages be better?

Reply:
Yeah we considered this but since we wanted to focus more on functionality and honing certain skills we opted for the single page application